src/WebJobs.Script.WebHost/Diagnostics/LinuxEventGenerator.cs

```diff
@@ -12,16 +12,23 @@ public abstract class LinuxEventGenerator : IEventGenerator
         public static readonly string EventTimestampFormat = "MM/dd/yyyy hh:mm:ss.fff tt";
 
         // These names should match the source file names for fluentd
-        public static readonly string FunctionsLogsCategory = "functionslogs";
+        public static readonly string FunctionsLogsCategory = "functionslogsv2";
         public static readonly string FunctionsMetricsCategory = "functionsmetrics";
         public static readonly string FunctionsDetailsCategory = "functionsdetails";
         public static readonly string FunctionsExecutionEventsCategory = "functionexecutionevents";
 
         internal static string NormalizeString(string value)
         {
-            // need to remove newlines for csv output
+            // Need to remove newlines for csv output
             value = value.Replace(Environment.NewLine, " ");
 
+            // Need to replace double quotes with single quotes as
+            // our regex query looks at double quotes as delimeter for
+            // individual column
+            // TODO: Once the regex takes into account for quotes, we can
+            // safely remove this
+            value = value.Replace("\"", "'");
```

Comment:
Pretty big nit - The comment states we can safely remove this once the regex is updated, but as we discussed before, that's not 100% accurate as it will result in a change in the log output. Just something we need to consider.

Reply:
That is a good point. The comment is misleading. I will update that separately.